packages/gateway/src/constants.ts

```diff
@@ -67,3 +67,8 @@ export const CONFIG_SYSTEM_TOKEN_EXPIRY_SECONDS = process.env
   : 600
 export const DEFAULT_TIMEOUT = 600000
 export const MINIO_BUCKET = process.env.MINIO_BUCKET || 'ocrvs'
+
+export const OIDP_BASE_URL = process.env.NATIONAL_ID_OIDP_BASE_URL!
+/** Base64 encoded RS256 JSON Web Key */
+export const OIDP_CLIENT_PRIVATE_KEY =
+  process.env.NATIONAL_ID_OIDP_CLIENT_PRIVATE_KEY!
```

Comment:
Would these variables be always provided?

Reply:
Yes, in case the integration is enabled. So they can be empty yes, but I think the integration can assume that they exists. Should we add some kind of check that they exist to the endpoint?

Reply:
I removed the `!` from here and added a null check to the file that uses these constants.